server/middlewares/checkUser.js

```diff
@@ -46,5 +46,35 @@ class checkUser {
       });
     }
   }
+
+  /**
+   * This method Checks if User is sucessfully authenticated with google.
+   * @param {object} req The user's request.
+   * @param {object} res The response.
+   * @param {Function} next pass to next function
+   * @returns {object} The status and some data of the user.
+   */
+  static async isAuthWithGoogle(req, res, next) {
+    passport.authenticate('google', { failureRedirect: '/google', session: false }, (err, user) => {
+      if (err) { return next(err); }
+      req.user = user;
+      return next();
+    })(req, res, next);
+  }
+
+  /**
+     * This method Checks if User is sucessfully authenticated with google.
```

Comment:
Update the comment. This is for Facebook instead.

Reply:
Feedback Implemented!